docs/components/board/customlinux.md

```diff
@@ -0,0 +1,313 @@
+      "line_number": 173,
+      "pwm_id": -1
+    },
+    {
+      "name": "13",
+      "device_name": "gpiochip0",
+      "line_number": 152,
+      "pwm_id": -1
+    },
+    {
+      "name": "15",
+      "device_name": "gpiochip0",
+      "line_number": 94,
+      "pwm_id": -1
+    },
+    {
+      "name": "19",
+      "device_name": "gpiochip0",
+      "line_number": 163,
+      "pwm_id": -1
+    },
+    {
+      "name": "21",
+      "device_name": "gpiochip0",
+      "line_number": 161,
+      "pwm_id": -1
+    },
+    {
+      "name": "23",
+      "device_name": "gpiochip0",
+      "line_number": 164,
+      "pwm_id": -1
+    },
+    {
+      "name": "27",
+      "device_name": "gpiochip0",
+      "line_number": 82,
+      "pwm_id": -1
+    },
+    {
+      "name": "29",
+      "device_name": "gpiochip0",
+      "line_number": 98,
+      "pwm_id": -1
+    },
+    {
+      "name": "31",
+      "device_name": "gpiochip0",
+      "line_number": 12,
+      "pwm_id": -1
+    },
+    {
+      "name": "33",
+      "device_name": "gpiochip0",
+      "line_number": 101,
+      "pwm_id": -1
+    },
+    {
+      "name": "35",
+      "device_name": "gpiochip0",
+      "line_number": 171,
+      "pwm_id": -1
+    },
+    {
+      "name": "37",
+      "device_name": "gpiochip0",
+      "line_number": 169,
+      "pwm_id": -1
+    },
+    {
+      "name": "8",
+      "device_name": "gpiochip0",
+      "line_number": 115,
+      "pwm_id": -1
+    },
+    {
+      "name": "10",
+      "device_name": "gpiochip0",
+      "line_number": 121,
+      "pwm_id": -1
+    },
+    {
+      "name": "12",
+      "device_name": "gpiochip0",
+      "line_number": 170,
+      "pwm_id": -1
+    },
+    {
+      "name": "16",
+      "device_name": "gpiochip0",
+      "line_number": 165,
+      "pwm_id": -1
+    },
+    {
+      "name": "18",
+      "device_name": "gpiochip0",
+      "line_number": 1,
+      "pwm_id": -1
+    },
+    {
+      "name": "22",
+      "device_name": "gpiochip0",
+      "line_number": 2,
+      "pwm_id": -1
+    },
+    {
+      "name": "24",
+      "device_name": "gpiochip0",
+      "line_number": 162,
+      "pwm_id": -1
+    },
+    {
+      "name": "26",
+      "device_name": "gpiochip0",
+      "line_number": 0,
+      "pwm_id": -1
+    },
+    {
+      "name": "28",
+      "device_name": "gpiochip0",
+      "line_number": 83,
+      "pwm_id": -1
+    },
+    {
+      "name": "32",
+      "device_name": "gpiochip0",
+      "line_number": 97,
+      "pwm_id": -1
+    },
+    {
+      "name": "36",
+      "device_name": "gpiochip0",
+      "line_number": 151,
+      "pwm_id": -1
+    },
+    {
+      "name": "38",
+      "device_name": "gpiochip0",
+      "line_number": 174,
+      "pwm_id": -1
+    },
+    {
+      "name": "40",
+      "device_name": "gpiochip0",
+      "line_number": 172,
+      "pwm_id": -1
+    }
+  ]
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+The following attributes are available for pin mappings:
```

Comment:
```suggestion
The following fields are available for pin mappings:
```
Would say this or parameters as it's not exactly the same thing as json component attributes right? Not 100% sure

Reply:
parameters probably even better
```suggestion
The following parameters are available for pin mappings:
```